Code/GraphMol/ForceFieldHelpers/CrystalFF/torsionPreferences_macrocycles.in

```diff
@@ -9,9 +9,9 @@
  */
 const std::string torsionPreferencesMacrocycles =
   // ----- experimental ring amide bond patterns
-  "[C:1][C;r{9-}:2](=O)@;-[NX3H0;r{9-}:3][CX4H1:4] 1 0.0 -1 8.0 1 0.0 1 0.0 1 0.0 1 0.0\n"
-  "[C:1][C;r{9-}:2](=O)@;-[NX3H1;r{9-}:3][CX4H1:4] 1 0.0 -1 8.0 1 0.0 1 0.0 1 0.0 1 0.0\n"
-  "[C:1][C;r{9-}:2](=O)@;-[NX3H0;r{9-}:3][CX4H2:4] 1 0.0 -1 8.0 1 0.0 1 0.0 1 0.0 1 0.0\n"
+  "[C:1][C;r{9-}:2](=O)@;-[NX3H0;r:3][CX4H1:4] 1 0.0 -1 8.0 1 0.0 1 0.0 1 0.0 1 0.0\n"
```

Comment:
The changes in this file are not mentioned in the PR description. Please add some note about them there.